replications/service.go

```diff
@@ -264,6 +267,26 @@ func (s service) DeleteReplication(ctx context.Context, id platform.ID) error {
 	return nil
 }
 
+func (s service) DeleteBucketReplications(ctx context.Context, localBucketID platform.ID) error {
+	s.store.Mu.Lock()
+	defer s.store.Mu.Unlock()
+
+	q := sq.Delete("replications").Where(sq.Eq{"local_bucket_id": localBucketID}).Suffix("RETURNING id")
+	query, args, err := q.ToSql()
+	if err != nil {
+		return err
+	}
+
+	var deleted []string
+	if err := s.store.DB.SelectContext(ctx, &deleted, query, args...); err != nil {
+		return err
+	}
+	s.log.Debug("Deleted all replications for local bucket",
```

Comment:
Do the other methods of this service use the logging middleware? Or is that only done through the server? Seems a little odd only seeing logging done for this one particular method but if the rest is done through the server that makes sense.

Reply:
The logging middleware logs the total time taken for each method, but this is the only one with other logs right now. My thinking was that the behavior of the other methods is pretty "obvious" based on the inputs & outputs, but this method could have some confusing behavior worth logging (i.e. "where did my replication go???" after deleting a bucket)